index.bs

```diff
@@ -1647,14 +1662,27 @@ To <dfn>serialize an integer</dfn>, represent it as a string of the shortest pos
 Issue: This would ideally be replaced by a more descriptive algorithm in Infra. See
 <a href="https://github.com/whatwg/infra/issues/201">infra/201</a>
 
+<h3 id="serialize-destinations">Serialize attribution destinations</h3>
+
+To <dfn>serialize [=event-level report/attribution destinations=]</dfn> |destinations|, run the following steps:
+
+1. [=Assert=]: |destinations| does not [=set/contain=] the [=opaque origin=].
+1. [=Assert=]: |destinations| is not [=set/empty=].
+1. Let |destinationStrings| be a [=list=]
+1. [=list/iterate|For each=] |destination| in |destinations|:
+    1. [=list/append=] |destination| <a href="https://html.spec.whatwg.org/multipage/origin.html#serialization-of-a-site">serialized</a> to |destinationStrings|
+1. If |destinationStrings|'s [=set/size=] is equal to 1, return |destinationStrings|[0].
+1. Return the result of running [=serialize an infra value to JSON bytes=] on |destinationStrings|.
```

Comment:
Do we actually want to serialize this here? It seems like the final report body should be:

```json
{
  "attribution_destination": ["a", "b", "c"]
}
```
or
```json
{
  "attribution_destination": "a"
}
```
rather than
```json
{
  "attribution_destination": "\"a\",\"b\",\"c\""
}
```